tests/Tests/Fixture/patients.json

```diff
@@ -0,0 +1,201 @@
+[
+	{
```

Comment:
In the patient file I'm omitting the id and pid, since the `id` is assigned via an auto-incrementing database column and the pid is "allocated" using a database query.

Reply:
What also neat about this (I think at least) is that could also run this data through api automated testing (when we have support for that)